test/distrib/gcf/python/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shellcheck cant find the included file.
+# shellcheck disable=SC1091
+source common.sh
```

Comment:
It might be a good idea to change directory to ensure the work directory is as expected. Like https://github.com/grpc/grpc/blob/master/tools/run_tests/helper_scripts/build_python.sh#L19

Reply:
True. Done.